OloEditor/src/Panels/ContentBrowserPanel.cpp

```
@@ -99,27 +296,237 @@ namespace OloEngine
         ImGui::End();
     }
 
-    Ref<Texture2D>& ContentBrowserPanel::GetFileIcon(const std::filesystem::path& filepath)
+    void ContentBrowserPanel::DrawCreateMenu()
```

Clang (14.0.6)

[warning] method 'DrawCreateMenu' can be made static

(readability-convert-member-functions-to-static)